### DISCUSSION TITLE
Any interest in Preact support?

### DISCUSSION THREAD

**Discussion by panthershark** [tailwindlabs/heroicons#696] [Help] [answered]
I'm using preact and would like to avoid the preact compat layer. Is supporting Preact palatable to the team that maintains this library?  If so, I could probably assist with the implementation.

Thanks for making all of these great svgs. Please lmk if this is something y'all would support.

**Comment by ekwoka** [comment, marked as answer]
@panthershark I've just released a package that has ALL of the icons as preact components, with `h` as the JSX factory (you can even choose a different one if you want, as it's all raw JSX) and types.

https://www.npmjs.com/package/preact-heroicons

I needed them working well with Preact, typescript, and being treeshakable, and that didn't seem to exist.

**Comment by panthershark** [comment]
Awesome.  Thanks for creating this @ekwoka!  

I'll try it out very soon. :)
